pytest.ini

```diff
@@ -0,0 +1,11 @@
+[pytest]
```

Comment:
This configuration can go into pyproject.toml to cut down on the number of files

Reply:
Well, yes, pytest.ini and tox.ini can be easily moved into the pyproject.toml file, it can be done in the next release when CI/CD be more refined.